player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -634,6 +640,9 @@ public void onSkipClick() {
     public void onOpenClick() {
 
         VASTLog.v(TAG, "onOpenClick");
+        if (mBannerBitmap != null) {
+            mIsBannerSet = true;
+        }
```

Comment:
this fix should be approached from a different side, this set of booleans is not really going to help with the solution in the long therm, just a quick fix